lib/generators/logidze/install/templates/migration.rb.erb

```diff
@@ -205,6 +244,11 @@ class <%= @migration_class_name %> < ActiveRecord::Migration<%= ActiveRecord::VE
             IF history_limit IS NOT NULL AND history_limit = size THEN
               NEW.log_data := logidze_compact_history(NEW.log_data);
             END IF;
+
+            IF debounce_time IS NOT NULL THEN
+              NEW.log_data := logidze_merge_recent(NEW.log_data, debounce_time);
```

Comment:
I think, it's better to handle debounce before adding a new version instead of adding a new first version first and than merging it with the previous.

So, on line 231 we can do:

```
IF debounce_time IS NOT NULL AND version->>'ts'::bigint - NEW.log_data#>'{h,-1,ts}')::text::bigin <= debounce_time
  # merge new version with the previous one
  version = logidze_merge_version(NEW.log_data#>'{h,-1}', version)
  # remove the previous version from log
  NEW.log_data = jsonb_set(
            NEW.log_data,
            '{h}',
             NEW.log_data->'h' - (size -1 )::text
          );
END
```

Reply:
thanks for the brilliant suggestion, I made the change accordingly and added some tests